projects/plugins/jetpack/_inc/client/components/module-settings/connect-module-options.jsx

```diff
@@ -22,8 +22,8 @@ import {
  * High order component that connects to Jetpack modules'options
  * redux state selectors and action creators.
  *
- * @param {React.Component} Component The component to be connected to the state
- * @return {[React.Component]} The component with some props connected to the state
+ * @param {import('react').Component} Component - The component to be connected to the state
+ * @return {[import('react').Component]} The component with some props connected to the state
```

Comment:
Should there be a React import in this file so we don't have to do doc gymnastics?

Reply:
I thought it would complain, but now that I try it it works fine. 3b45678063